docs/user/adding-pdf-annotations.md

```diff
@@ -317,16 +317,22 @@ you can use the {py:class}`Highlight <pypdf.annotations.Highlight>`:
 ```python
 from pypdf import PdfReader, PdfWriter
 from pypdf.annotations import Highlight
+from pypdf.generic import ArrayObject
+from pypdf.generic._base import FloatObject
```

Comment:
Please avoid importing from the private namespace and use the public one from https://github.com/py-pdf/pypdf/blob/38795f5cb23f1950c65a2928763747230db84374/pypdf/generic/__init__.py#L39 instead.

Reply:
like this: `from pypdf.generic import ArrayObject, FloatObject` ?